### ISSUE TITLE
websocketserver.hpp is missing from LIBDATACHANNEL_HEADERS

### ISSUE THREAD

**Issue by asgeir** [paullouisageneau/libdatachannel#457]
`websocketserver.hpp` is missing from `LIBDATACHANNEL_HEADERS` so it doesn't get installed. This breaks builds when `rtc.hpp` tries to include it.

```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1a6434e..536b65f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -86,6 +86,7 @@ set(LIBDATACHANNEL_HEADERS
        ${CMAKE_CURRENT_SOURCE_DIR}/include/rtc/rtp.hpp
        ${CMAKE_CURRENT_SOURCE_DIR}/include/rtc/track.hpp
        ${CMAKE_CURRENT_SOURCE_DIR}/include/rtc/websocket.hpp
+       ${CMAKE_CURRENT_SOURCE_DIR}/include/rtc/websocketserver.hpp
        ${CMAKE_CURRENT_SOURCE_DIR}/include/rtc/rtppacketizationconfig.hpp
        ${CMAKE_CURRENT_SOURCE_DIR}/include/rtc/rtcpsrreporter.hpp
        ${CMAKE_CURRENT_SOURCE_DIR}/include/rtc/rtppacketizer.hpp
```

**Comment by paullouisageneau** [comment]
Arf, indeed, thanks for reporting! Fixed in https://github.com/paullouisageneau/libdatachannel/pull/458

**Comment by paullouisageneau** [comment]
@Nemirtingas This is shipped in v0.14.2, I don't know if it impacts vcpkg.

**Comment by Nemirtingas** [comment]
It does impact only if the user tries to use it, I had no issue building the client websocket and the full library.
